One-Foot-In/ofi-alumni
# Allow Alumni to curate opportunities and students to bookmark them

Alumni are able to curate opportunities (add aa description, tag interests, and an optional relevant link) and submit the opportunity. The opportunity then shows up in the Poll Carousel in the NewsFeed for students in the same school. Students in the same country or anywhere else may see the opportunity if there are at a sufficiently high access level (in a supernetwork)

Students can bookmark or pass on the opportunity. Alumni get a little star on any opportunity that has been bookmarked.

![image](https://user-images.githubusercontent.com/31665569/104464527-081a1000-5581-11eb-8dd2-51c2b1011487.png)
![image](https://user-images.githubusercontent.com/31665569/104466197-f0438b80-5582-11eb-9afb-b98822aed316.png)
Alumni's POV, opportunities subtab

![image](https://user-images.githubusercontent.com/31665569/104465099-b02fd900-5581-11eb-9a97-3fd7c724e335.png)
Student POV, bookmark or pass on opportunity

![image](https://user-images.githubusercontent.com/31665569/104465706-5380ee00-5582-11eb-9421-cf34580a6a6d.png)
Student POV, opportunities subtab

Test Steps:

1. Work with a single school context. Using mongo shell when running app locally, find an alumnus and a student who share the same school.
2. Log in as alumnus, and curate an opportunity
3. Log in as a student and confirm you see that opportunity in the Poll Carousel.
4. Bookmark the opportunity, and confirm the opportunity shows up in your Workspaces > Opportunities tab
5. Log back in as curating alumnus and confirm you can see a star on the opportunity just bookmarked
6. Confirm you can delete an opportunity as an alumnus
7. Confirm you can pass on an opportunity as a student, or delete a bookmarked opportunity
